chainer/functions/math/sparse_matmul.py

```diff
@@ -23,10 +23,16 @@ def _coo_matmul(sp_data, sp_row, sp_col, sp_shape, dn, transa, transb, transc,
         A_row = sp_col
         A_col = sp_row
         A_shape = (sp_shape[1], sp_shape[0])
+        A_order = sp_order
+        if sp_order is 'C':
+            A_order = 'F'
+        elif sp_order is 'F':
+            A_order = 'C'
```

Comment:
How about replacing the line above (`A_order = sp_order`) with an `else:` here, avoiding possible reassignment. Is it also the case that it's always `None`? Then we could be explicit about that too.